### ISSUE TITLE
小小小小草！+Paddle分布式训练文档体验评估

### ISSUE THREAD

**Issue by hatimwen** [PaddlePaddle/docs#4879]
### 任务类型

单机多卡

### 项目环境

AI Studio

### 认领任务

【NLP】IMDB 数据集使用BOW网络的文本分类：https://www.paddlepaddle.org.cn/documentation/docs/zh/practices/nlp/imdb_bow_classification.html

### 1.导入分布式训练需要的依赖包

- 完成情况（成功/不成功) ：成功
- 遇到问题：无
- 解决方法（无法解决需注明)：无
- 对应文档是否描述清晰（如不清晰，需注明问题并提供文档链接）：文档清晰

### 2.初始化分布式训练环境

- 完成情况（成功/不成功) ：成功
- 遇到问题：无
- 解决方法（无法解决需注明)：无
- 对应文档是否描述清晰（如不清晰，需注明问题并提供文档链接）：文档清晰

### 3.设置分布式训练需要的优化器

- 完成情况（成功/不成功) ：成功
- 遇到问题：无
- 解决方法（无法解决需注明)：无
- 对应文档是否描述清晰（如不清晰，需注明问题并提供文档链接）：文档清晰

### 4.数据集拆分

- 完成情况（成功/不成功) ：成功
- 遇到问题：无
- 解决方法（无法解决需注明)：无
- 对应文档是否描述清晰（如不清晰，需注明问题并提供文档链接）：文档清晰

### 5.构建训练代码

- 完成情况（成功/不成功) ：成功
- 遇到问题：无
- 解决方法（无法解决需注明)：无
- 对应文档是否描述清晰（如不清晰，需注明问题并提供文档链接）：文档清晰

### 6.单机多卡分布式训练

- 完成情况（成功/不成功) ：成功
- 遇到问题：无
- 解决方法（无法解决需注明)：无
- 对应文档是否描述清晰（如不清晰，需注明问题并提供文档链接）：文档清晰

### 7.多机多卡分布式训练

- 完成情况（成功/不成功) ：成功
- 遇到问题：无
- 解决方法（无法解决需注明)：无
- 对应文档是否描述清晰（如不清晰，需注明问题并提供文档链接）：文档清晰

### 其他问题或建议

无，流程很清晰。

### 相关代码

https://aistudio.baidu.com/aistudio/projectdetail/4135829?contributionType=1

### 认领任务名称

Paddle2.0：基于Seq2Seq的文本加法：https://aistudio.baidu.com/aistudio/projectdetail/1087885?channelType=0

### 1.导入分布式训练需要的依赖包

- 完成情况（成功/不成功) ：成功
- 遇到问题：无
- 解决方法（无法解决需注明)：无
- 对应文档是否描述清晰（如不清晰，需注明问题并提供文档链接）：文档清晰

### 2.初始化分布式训练环境

- 完成情况（成功/不成功) ：成功
- 遇到问题：无
- 解决方法（无法解决需注明)：无
- 对应文档是否描述清晰（如不清晰，需注明问题并提供文档链接）：文档清晰

### 3.设置分布式训练需要的优化器

- 完成情况（成功/不成功) ：成功
- 遇到问题：无
- 解决方法（无法解决需注明)：无
- 对应文档是否描述清晰（如不清晰，需注明问题并提供文档链接）：文档清晰

### 4.数据集拆分

- 完成情况（成功/不成功) ：成功
- 遇到问题：无
- 解决方法（无法解决需注明)：无
- 对应文档是否描述清晰（如不清晰，需注明问题并提供文档链接）：文档清晰

### 5.构建训练代码

- 完成情况（成功/不成功) ：成功
- 遇到问题：原项目PaddlePaddle版本为2.0.0-rc0，其CE损失`nn.functional.cross_entropy(out, label)`中的模型估计输出`out`做了[0,2,1]换序，在PaddlePaddle版本2.3.0中会报错。
- 解决方法（无法解决需注明)：在PaddlePaddle版本2.3.0中out不需要转置换序处理。
- 对应文档是否描述清晰（如不清晰，需注明问题并提供文档链接）：文档清晰

### 6.单机多卡分布式训练

- 完成情况（成功/不成功) ：成功
- 遇到问题：无
- 解决方法（无法解决需注明)：无
- 对应文档是否描述清晰（如不清晰，需注明问题并提供文档链接）：文档清晰

### 7.多机多卡分布式训练

- 完成情况（成功/不成功) ：成功
- 遇到问题：无
- 解决方法（无法解决需注明)：无
- 对应文档是否描述清晰（如不清晰，需注明问题并提供文档链接）：文档清晰

### 其他问题或建议

无，流程很清晰。

### 相关代码

https://aistudio.baidu.com/aistudio/projectdetail/4137775
